includes/Core/Conversion_Tracking/Conversion_Event_Providers/WooCommerce.php

```diff
@@ -91,10 +95,9 @@ protected function get_wgai_event_names() {
 			return array();
 		}
 
-		$settings    = get_option( 'woocommerce_google_analytics_settings' );
-		$event_names = array();
```

Comment:
This variable is redundant because it is defined again lower down (where it is actually used) and is essentially never used.